File: bench/bench_regex.pl
my $text = "The quick brown fox jumps over the lazy dog";
my $count = 0;
for (my $i = 0; $i < 100_000; $i = $i + 1) {
    if ($text =~ /(\w+)\s+(\w+)$/) {
        $count = $count + 1;
    }
}
print $count, "\n";

File: bench/bench_regex.jl
function main()
    text = "The quick brown fox jumps over the lazy dog"
    pat = r"(\w+)\s+(\w+)$"
    count = 0
    for i in 1:100_000
        if occursin(pat, text)
            count += 1
        end
    end
    println(count)
end
main()
